clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/IngestionThrottler.java

```diff
@@ -104,22 +104,18 @@ public IngestionThrottler(
             "current_version_non_aa_wc_leader_records_count",
             false,
             EventThrottler.BLOCK_STRATEGY));
-    this.poolTypeRecordThrottlerMap.put(
-        ConsumerPoolType.NON_CURRENT_VERSION_AA_WC_LEADER_POOL,
-        new EventThrottler(
-            serverConfig.getNonCurrentVersionAAWCLeaderQuotaRecordsPerSecond(),
-            serverConfig.getKafkaFetchQuotaTimeWindow(),
-            "non_current_version_aa_wc_leader_records_count",
-            false,
-            EventThrottler.BLOCK_STRATEGY));
-    this.poolTypeRecordThrottlerMap.put(
-        ConsumerPoolType.NON_CURRENT_VERSION_NON_AA_WC_LEADER_POOL,
-        new EventThrottler(
-            serverConfig.getNonCurrentVersionNonAAWCLeaderQuotaRecordsPerSecond(),
-            serverConfig.getKafkaFetchQuotaTimeWindow(),
-            "non_current_version_non_aa_wc_leader_records_count",
-            false,
-            EventThrottler.BLOCK_STRATEGY));
+    VeniceAdaptiveIngestionThrottler adaptiveIngestionThrottler = new VeniceAdaptiveIngestionThrottler(
```

Comment:
Could you wrap this with the newly added config? If the config is disabled, we won't use the new throttler. This is to make sure our production does not directly embrace this change.